authority-portal-frontend/src/app/pages/catalog-page/catalog-page/catalog-page.component.ts

```diff
@@ -113,6 +114,12 @@ export class CatalogPageComponent implements OnInit, OnDestroy {
         const initialMdsIds = mdsId ? [mdsId] : undefined;
         this.store.dispatch(new CatalogPage.Reset(initialMdsIds));
       });
+
+    const params = this.route.firstChild?.snapshot.params;
+    if (params) {
+      console.log('PARAMS ' + params['connectorId'] + ' ' + params['assetId']);
+      this.openDataOfferDialog(params['assetId'], params['connectorId']);
+    }
```

Comment:
look at the ngOnInit(), let's make this a function, too. "this.openDataOfferDetailDialogOnceFromUrl()"